tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -549,6 +549,7 @@ LogicalResult LowerHLOToLLVM(ModuleOp m, const DISCLoweringOptions& options) {
     pm.addNestedPass<FuncOp>(disc_ral::createForLoopUnrollInterleavePass());
   }
   pm.addNestedPass<FuncOp>(arith::createArithExpandOpsPass());
+  pm.addNestedPass<FuncOp>(disc_ral::createDiscBF16ExpansionPass());
```

Comment:
nit：better to add some comments to explain that this part of code can be removed after we rebase to the latest llvm head.